doc/release-notes/release-3.5.sgml

```diff
@@ -98,8 +98,8 @@ Most user-facing changes are reflected in squid.conf (see below).
 
 
 <sect1>Support named services
-<p>Details at <url url="http://wiki.squid-cache.org/MultipleInstances">.
-<p>Terminology details at <url url="http://wiki.squid-cache.org/Features/SmpScale#Terminology">.
+<p>Details at <url url="https://wiki.squid-cache.org/MultipleInstances">.
+<p>Terminology details at <url url="https://wiki.squid-cache.org/Features/SmpScale#Terminology">.
```

Comment:
```suggestion
<p>Terminology details at <url url="https://wiki.squid-cache.org/Features/SmpScale#terminology">.
```

Again, I do not think we should be updating old release notes, even if the old URLs stop working, but I am not blocking this PR because of those updates.

Reply:
Done in [9e227de](https://github.com/squid-cache/squid/pull/1448/commits/9e227deea244a6b3e0a873a8b8f1e741c2f6fdfe).


FYI, I am including these so all the master branch content is correct. Plan is to drop these files once distros stop packaging the v3 series.